modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/autocomplete/autocomplete.js

```diff
@@ -183,6 +183,25 @@
 				}
 			});
 
+			// What follows is an algorithm to properly select the query from the
+			// last detected trigger. Because of the HTML structure, it's not as
+			// straightforward as a text search.
+			//
+			// If triggerIndex === -1, the trigger is not in the current html node
+			// element. Thus, we walk the DOM tree upwards until we find it,
+			// constructing the query as we visit the nodes in the tree.
+			//
+			// If triggerIndex > 1, we found the trigger in a longer text sequence.
+			// We check if the char to the trigger is a space (' ' or nbsp;) by
```

Comment:
"if the char to the trigger" → "if the char before the trigger"

`nbsp;` → `&nbsp;`